store/subscription.go

```diff
@@ -259,15 +271,15 @@ func (s *subscription) insert(ctx context.Context, events []*model.Transfer) (er
 			}
 
 			// Add fee if it's a ETH event
-			if f, ok := fees[gethCommon.BytesToHash(e.TxHash)]; ok && bytes.Equal(e.Address, model.ETHBytes) {
+			if f, ok := fees[gethCommon.BytesToHash(e.TxHash)]; bytes.Equal(e.Address, model.ETHBytes) && ok {
```

Comment:
can you move `ok` to before comparing with ETHBytes? i changed it to save a comparison if it's not in `fees`

Reply:
ok, my fault in rebase

Reply:
done, my fault in rebase